lib/ex_doc/formatter/markdown/templates/summary_template.eex

```diff
@@ -0,0 +1,15 @@
+### <%= name %>
```

Comment:
Do we need summaries for Markdown?

I think we only need summaries if we are going to emit a different markdown file per function/macro/callback/type. Which would be helpful to help us implement the copy feature...